apps/dialtone-documentation/docs/components/modal.md

```diff
@@ -143,6 +143,8 @@ showHtmlWarning />
 
 ### Fixed header and footer
 
+This is the default behavior that adds the scroll automatically in the modal content and leaves the header and footer fixed.
+
 <code-well-header>
   <example-modal kind="fixed" />
 </code-well-header>
```

Comment:
This variant desperately needs a custom scrollbar. There's zero affordance that it's a scrollable area.

I know that's outside this tasks's scope. Can you log an issue and see if we can get that out sooner? An epic should probably be created to collect all the DT components and recipes that should a custom scrollbar.

![image](https://github.com/user-attachments/assets/20f531e0-e3ac-45b2-842d-95fe9ce485a9)

Reply:
I logged the ticket here: https://dialpad.atlassian.net/browse/DLT-1983

Reply:
Thanks, I've chucked it in a [new Epic](https://dialpad.atlassian.net/browse/DLT-1984) alongside other ones I've identified.